app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -16,7 +16,7 @@
     <div id="ellipsis-menu-dropdown-<%= article.id %>" class="crayons-dropdown top-100 right-0 z-10 align-left js-ellipsis-menu-dropdown">
       <%= form_for(article, method: :patch, remote: true, authenticity_token: true, html: { "data-type": "json", class: "p-0 js-archive-toggle" }) do |f| %>
         <%= f.hidden_field :archived, value: !article.archived %>
-
+        <%= f.hidden_field :from_dashboard, value: 1 %>
```

Comment:
I added `from_dashboard` just in case here, this partial is only used for published posts, and the bug is only related to the scheduled, but still we don't pass `published_at` here, so it makes sense to make sure that we don't intend to nullify it.